app/jenkins-for-jira-ui/src/components/ConnectionPanel/ConnectionPanel.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from 'react';
+import { cx } from '@emotion/css';
+import { ConnectionPanelMain } from './ConnectionPanelMain';
+import { ConnectionPanelTop } from './ConnectionPanelTop';
+import { connectionPanelContainer } from './ConnectionPanel.styles';
+import { ConnectedState } from '../StatusLabel/StatusLabel';
+
+const ConnectionPanel = (): JSX.Element => {
+	// TODO - remove temp state and define pending/duplicate/connected state from data
+	const [connectedState, setConnectState] = useState<ConnectedState>(ConnectedState.PENDING);
+
+	useEffect(() => {
+		// TODO - update this based on data
+		setConnectState(ConnectedState.DUPLICATE);
```

Comment:
Will update this later to set state based on the data we get. In the meantime, feel free to change this from DUPLICATE/CONNECTED/PENDING to see the diff status labels.